pymodels/BO_V06_01/accelerator.py

```diff
@@ -25,8 +28,23 @@ def create_accelerator(
     return accelerator
 
 
+lattice_version = 'BO_V06_01'
+_path, _ver = _get_path('pymodels')
+if _is_git_repo(_path):
```

Comment:
I think this creation of a standardized tag string should be a function of `mathphys` too.
Note how you had to repeat the code along all accelerators and how you had to import a lot of methods from `mathphys`.

Reply:
Sure, good idea. I implemented the method `get_package_string` in `mathphys`: https://github.com/lnls-fac/mathphys/pull/25/commits/cb1a8d36b9689df2336e0c559b70c80a108b08c0. Thanks!